### PR TITLE
Add content on custom tracing port

### PR DESCRIPTION
**Pull request by shavidissa**
Add content on custom tracing port

### CODE REVIEW

**Comment by shavidissa** [comment]
The doc link is broken for now until the changes get pushed to mater.

File: README.md [hunk 1]
```diff
@@ -197,6 +197,25 @@ WavefrontSender wavefrontSender = new WavefrontClient.Builder(proxyServerWithPor
         build(); // Returns a WavefrontClient
 ```
 
+When you use a Sender SDK, you won’t see span-level RED metrics by default unless you use the Wavefront proxy and define a custom tracing port (`tracingPort`). See [Instrument Your Application with Wavefront Sender SDKs](https://www.wavefront.com/tracing_instrumenting_frameworks.html/#instrument-your-application-with-wavefront-sender-sdks) for details.
```
Comment (srujann) [inline README.md:200]: this is relevant only for spans. Can you add a subsection title that this is related to spans?

File: README.md [hunk 2]
```diff
@@ -190,6 +190,9 @@ WavefrontSender wavefrontSender = wavefrontClientFactory.getClient();
 ```java
 // Proxy based ingestion
 WavefrontSender wavefrontSender = new WavefrontClient.Builder(proxyServerWithPort).
+        tracingPort(tracing port). // the same port as the customTracingListenerPorts configured in the wavefront proxy
```
Comment (srujann) [inline README.md:193]: `customTracingListenerPorts` should be used only when the sender is not used with opentracing sdk or any other app level sdk. Can you please clarify that? same below.

Comment (shavidissa) [reply README.md]: Updated. Does that sound better?

**Pull request merged**